source/jormungandr/jormungandr/instance_manager.py

```diff
@@ -341,3 +342,24 @@ def regions(self, region=None, lon=None, lat=None, request_id=None):
             resp_dict['region_id'] = key_region
             response['regions'].append(resp_dict)
         return response
+
+    def get_all_available_instances(self, user):
```

Comment:
This function should be cached.
As discussed with @pbench if we can remove the condition as described in https://github.com/hove-io/navitia/blob/17d38284168f39d3f888fb77d19f2a9ec9083281/source/jormungandr/jormungandr/authentication.py#L255
we can re-user the function authentification.get_all_available_instances or add a simplified and cached function only to be used for this purpose.

Reply:
Thanks ! I added a cache